test/cmocka/src/audio/volume/CMakeLists.txt

```diff
@@ -15,6 +15,8 @@ add_library(audio_for_volume STATIC
 	${PROJECT_SOURCE_DIR}/src/audio/volume/volume.c
 	${PROJECT_SOURCE_DIR}/src/audio/volume/volume_generic.c
 	${PROJECT_SOURCE_DIR}/src/audio/volume/volume_hifi3.c
+	${PROJECT_SOURCE_DIR}/src/audio/buffer.c
+	${PROJECT_SOURCE_DIR}/test/cmocka/src/notifier_mocks.c
```

Comment:
ditto for patch subject as per last subject coment.

Reply:
changed